config/data/drug_stock_config.yml

```diff
@@ -180,6 +180,7 @@ Punjab:
       '979467': 1
     hypertension_diuretic:
       new_patient_coefficient: 0.06
+      '316047': 0.5
```

Comment:
```suggestion
      '429503': 0.5
```

Reply:
It looks like the drugs in our data set are using `429503` _(hydrochlorothiazide 12.5 MG Oral Tablet)_ as the rxnorm_code, rather than  `316047` _(hydrochlorothiazide 12.5 MG)_ 

We should update any other references to `316047`as well